app/controllers/works_controller.rb

```diff
@@ -96,8 +98,10 @@ def resolve_ark
   end
 
   # GET /works/1/edit
+  # rubocop:disable Metrics/MethodLength
   def edit
     @work = Work.find(params[:id])
+    @work_decorator = WorkDecorator.new(@work, current_user)
```

Comment:
It does not seem that we actually using this variable in the method, do we need it?